examples/agents/ibm_granite_bee.ts

```diff
@@ -0,0 +1,125 @@
+import "dotenv/config.js";
+import { BeeAgent } from "bee-agent-framework/agents/bee/agent";
+import { ChatLLM, ChatLLMOutput } from "bee-agent-framework/llms/chat";
+import { getEnv, parseEnv } from "bee-agent-framework/internals/env";
+import { FrameworkError } from "bee-agent-framework/errors";
+import { TokenMemory } from "bee-agent-framework/memory/tokenMemory";
+import { WatsonXChatLLM } from "bee-agent-framework/adapters/watsonx/chat";
+import { OpenAIChatLLM } from "bee-agent-framework/adapters/openai/chat";
+import { OllamaChatLLM } from "bee-agent-framework/adapters/ollama/chat";
+import { IBMVllmChatLLM } from "bee-agent-framework/adapters/ibm-vllm/chat";
+import { IBMVllmModel } from "bee-agent-framework/adapters/ibm-vllm/chatPreset";
+import { OpenMeteoTool } from "bee-agent-framework/tools/weather/openMeteo";
+import { DuckDuckGoSearchTool } from "bee-agent-framework/tools/search/duckDuckGoSearch";
+import { Ollama } from "ollama";
+import OpenAI from "openai";
+import { z } from "zod";
+import * as process from "node:process";
+import fs from "node:fs";
+
+const Providers = {
+  WATSONX: "watsonx",
+  OLLAMA: "ollama",
+  OPENAI: "openai",
+  IBMVLLM: "ibmvllm",
+  IBMRITS: "ibmrits",
+} as const;
+type Provider = (typeof Providers)[keyof typeof Providers];
+
+function getChatLLM(provider?: Provider): ChatLLM<ChatLLMOutput> {
+  const LLMFactories: Record<Provider, () => ChatLLM<ChatLLMOutput>> = {
+    [Providers.OPENAI]: () =>
+      new OpenAIChatLLM({
+        modelId: getEnv("OPENAI_MODEL") || "ibm-granite/granite-3-8b-instruct",
+        parameters: {
+          temperature: 0,
+          max_tokens: 2048,
+        },
+      }),
```

Comment:
Granite is not available in OpenAI directly. Remove.

Reply:
Resolved in 213f963